backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/createFrom/po_from_so/impl/CreatePOFromSOsAggregationKeyBuilder.java

```diff
@@ -131,10 +131,10 @@ public String buildKey(final I_C_OrderLine salesOrderLine)
 		if (p_Vendor_ID > 0 && poPartnerRepoId != p_Vendor_ID)
 		{
 			final String msg = msgBL.getMsg(context.getCtx(),
-					MSG_VENDOR_MISMATCH,
-					new Object[] { salesOrder.getDocumentNo(), salesOrderLine.getLine(), poBPartnerRecord.getValue(), poBPartnerRecord.getName() });
+											MSG_VENDOR_MISMATCH,
+											new Object[] { salesOrder.getDocumentNo(), salesOrderLine.getLine(), poBPartnerRecord.getValue(), poBPartnerRecord.getName() });
 
-			loggable.addLog(msg);
+			Loggables.get().addLog(msg);
```

Comment:
please also add a "normal" static logger and the call Loggables.withLogger(logger, Level.DEBUG)